src/JSONPrinting.cpp

```diff
@@ -28,17 +24,19 @@ std::string getEntireMessage(const std::string &JSONMessage,
 }
 
 /// Receives deserialized flatbuffers message, truncated large arrays, formats
-/// it and prints it to the screen.
+/// it using Indent and prints it to the screen.
 ///
 /// \param JSONMessage
+/// \param Indent
```

Comment:
I assume this is number of characters of whitespace to use for indentation, but best to clarify in the comment.